lib/snapshot-list/snapshot-list__item--active.html

```diff
@@ -0,0 +1,20 @@
+<li>
+  <div class="snapshot-list__item--active">
+    <div class="snapshot-list__item__index">
+      <h4>{{ index }}</h4>
```

Comment:
I'm probably wrong but won't putting spaces in between the {{ }} actually put the spaces in there? Swear I've seen that before.